README.md

```diff
@@ -43,7 +41,7 @@ A non-zero exit code and error message will be returned for any failures:
     Tasks: TOP => syntax => syntax:manifests
     (See full trace by running task with --trace)
 
-Use in conjunction with lint and spec tests for Continuous Integration:
+Use in conjunction with lint [TODO: puppet-lint?] and spec tests [TODO: spec tests being general spec testing, and not a specific tool?] for Continuous Integration:
```

Comment:
puppet-lint: Yes.

spec: often in the context of modules, spec, rspec, and rspec-puppet are (incorrectly) used interchangeably. In this specific case, it is prompted by the name of the rake task (`:spec`) below, which makes it acceptable.